code/game/objects/items/weapons/weaponry.dm

```diff
@@ -166,16 +166,16 @@
 	icon_state = "switchblade"
 	desc = "A sharp, concealable, spring-loaded knife."
 	flags = CONDUCT
-	force = 20
+	force = 3
 	w_class = 2
-	throwforce = 15
+	throwforce = 5
 	throw_speed = 3
 	throw_range = 6
 	materials = list(MAT_METAL=12000)

```

Comment:
Isn't this, like, 6 sheets of metal? Can't 6 sheets of metal build a wall and a nice chair?